### ISSUE TITLE
Tail processes are not cleaned up after scan

### ISSUE THREAD

**Issue by mdvickst** [quipucords/rho#506]
# Opening a new issue
___

## Specify type:
- Bug

### Bug severity (if applicable):
- High

___

## Description:
Tail processes which produce logs still show up in ps after scan is complete. 
___

### Version of rho:
0.0.30

### Expected behavior:
They should close when rho is done

### Actual behavior:
processes live on

### Steps to reproduce:
Run Rho scan then once it is complete run ps -ef | grep tail

**Comment by chambridge** [comment]
Believe this to be a duplicate of https://github.com/quipucords/rho/issues/499 Just seen in a different context. Will clean up the thread handling with 499 and should resolve both issues.
